source/_docs/configuration/events.markdown

```diff
@@ -1,6 +1,11 @@
 ---
 title: "Events"
 description: "Describes all there is to know about events in Home Assistant."
+related:
+  - docs: /docs/automation/trigger/#event-trigger
+    title: Event triggers
+  - docs: /integrations/event/
+    title: Event integration
```

Comment:
well, the idea was to link topics that are related to each other. Core events are also events, no? You think maybe there should be an explainer section added?

Reply:
Yeah, I think it would be good to explain the difference.

Eg the event trigger is used for core events, not for the Event integration events.

The Event integration provides entities that trigger state change (core) events. A core event can represent more than state change events. It's a fundamental part of the Home Assistant backend. Core events are fired via the event bus in Home Assistant.

https://developers.home-assistant.io/docs/architecture/core

Reply:
<https://github.com/home-assistant/home-assistant.io/pull/34574>